package.json

```diff
@@ -66,6 +66,7 @@
     "resin-sdk": "^1.2.0",
     "resin-settings-client": "^1.0.0",
     "resin-vcs": "^1.2.0",
+    "tmp": "0.0.25",
```

Comment:
I think npm install --save uses a strict version for some reason on tmp, I had it happen to me as well.
Maybe that's why it it happened here too. Better use ^